### DISCUSSION TITLE
Consequences of uninstalling the extension

### DISCUSSION THREAD

**Discussion by TheAdminDavid** [hangy/azure-boards-estimate#226] [Q&A] [answered]
Hello,
I have a question regarding the uninstallation of Estimate.
We currently have this extension disabled but still installed.

What will happen if we uninstall it? Can this break something?
Or is there any loss in data with possible dependencies to other stuff?

Thanks for the help.

**Comment by hangy** [comment, marked as answer]
The only thing that could be deleted by uninstalling the extension is currently open estimation sessions. If you're worried about possibly losing the estimations, then there's no need to worry, because those are stored direcoty in the Work Items, not in a separate storage of this extension.
